lib/mayaUsd/ufe/UsdUndoCreateGroupCommand.h

```diff
@@ -26,7 +26,7 @@ namespace MAYAUSD_NS_DEF {
 namespace ufe {
 
 //! \brief UsdUndoCreateGroupCommand
-class MAYAUSD_CORE_PUBLIC UsdUndoCreateGroupCommand : public Ufe::CompositeUndoableCommand
+class MAYAUSD_CORE_PUBLIC UsdUndoCreateGroupCommand : public Ufe::InsertChildCommand
```

Comment:
UsdUndoCreateGroupCommand  now drives from InsertChildCommand and its insertedChild() should return the group sceneItem. 

This class also no longer need to drive from CompositeUndoableCommand and instead _groupCompositeCmd handles the command composition. 

This also makes it possible to have a clean undo()/redo() calls.